app/src/main/java/io/github/droidkaigi/confsched2017/view/customview/OverScrollLayout.java

```diff
@@ -138,11 +132,11 @@ public void onAnimationRepeat(Animator animation) {
                     });
             return;
         }
-        if (getY() != originalLocation.y) {
+        if (getY() != originalRect.top) {
```

Comment:
Oops, it's my mistake to insert null check at first if statement. 😇 
I think it is just need in onAnimationEnd.
IMO, dismissing view and restoring view animation should be performed regardless of whether listener is null or not.
Difference should be just performed callback or not.